lib/Item/ItemAccessibilitySupport.js

```diff
@@ -18,11 +18,39 @@ module.exports = {
 	/**
 	* @private
 	*/
+	handlers: {
```

Comment:
I'm concerned about adding handlers in the mixin as they'll stomp any existing handlers defined for these events. We could pursue a similar approach to Marquee and override dispatchEvent to handle these. If that's the route we take, I think we'll want to generalize that into a reusable spot (I'm thinking as another export to `enyo/dispatcher`) so both this and moonstone/Marquee can take advantage.